package.json

```diff
@@ -33,7 +33,8 @@
     "test:e2e:ci:bs": "yarn build && yarn build:apps && yarn test:e2e:bs",
     "test:compat:tsc": "node scripts/check-typescript-compatibility.js",
     "test:compat:ssr": "scripts/cli check_server_side_rendering_compatibility",
-    "rum-events-format:sync": "scripts/cli update_submodule && scripts/cli build_json2type && node scripts/generate-schema-types.js",
+    "rum-events-format:sync": "scripts/cli update_submodule && yarn rum-events-format:generate",
+    "rum-events-format:generate": "scripts/cli build_json2type && node scripts/generate-schema-types.js",
```

Comment:
I found it useful to generate locally the files I had checked-out in the `rum-events-format` submodule, mainly to confirm everything works as expected.
It's slightly different from `rum-events-format:sync` because there is no sync, on the file generation taken from the submodule.